homeassistant/components/tedee/config_flow.py

```diff
@@ -0,0 +1,100 @@
+"""Config flow for Tedee integration."""
+from typing import Any
+
+from pytedee_async import (
+    TedeeAuthException,
+    TedeeClient,
+    TedeeClientException,
+    TedeeLocalAuthException,
+)
+from pytedee_async.bridge import TedeeBridge
+import voluptuous as vol
+
+from homeassistant import config_entries, exceptions
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_HOST
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import CONF_LOCAL_ACCESS_TOKEN, DOMAIN, NAME
+
+
+async def validate_input(user_input: dict[str, Any]) -> bool:
+    """Validate the user input allows us to connect."""
+    pak = user_input.get(CONF_ACCESS_TOKEN, "")
+    host = user_input.get(CONF_HOST, "")
+    local_access_token = user_input.get(CONF_LOCAL_ACCESS_TOKEN, "")
+    tedee_client = TedeeClient(pak, local_access_token, host)
+
+    try:
+        await tedee_client.get_locks()
+    except (TedeeAuthException, TedeeLocalAuthException) as ex:
+        raise InvalidAuth from ex
+    except (TedeeClientException, Exception) as ex:
+        raise CannotConnect from ex
+    return True
+
+
+async def get_local_bridge(host: str, local_access_token: str) -> TedeeBridge:
+    """Get the serial number of the local bridge."""
+    tedee_client = TedeeClient(local_token=local_access_token, local_ip=host)
+    try:
+        return await tedee_client.get_local_bridge()
```

Comment:
If we can't connect to the device or have invalid auth, will this function behave the same as `tedee_client.get_locks()`? Why not use this function call for validation?